apis/database.js

```diff
@@ -80,9 +112,35 @@ export async function addLocalLocation(locationData) {
 
   const createdAt = moment.utc(locationObject.created).toISOString();
   await executeTransaction(
-    'insert into locations (key, coordinateKey, createdAt, resources, status, uploaded) values (?, ?, ?, ?, ?, ?)',
-    [key, key, createdAt, resourcesString, status, 0]
+    'insert into locations (key, coordinateKey, createdAt, resources, status, uploaded, updated) values (?, ?, ?, ?, ?, ?, ?)',
+    [key, key, createdAt, resourcesString, status, 0, 0]
   );
 
   return locationObject;
 }
+
+export async function createOrUpdateDatabase() {
+  const version = await SecureStore.getItemAsync(DATABASE_VERSION_KEY);
+
+  await createDatabases();
+
+  if (version === Constants.manifest.extra.localDatabaseVersion) {
+    return Promise.resolve();
```

Comment:
Style: You can `return` directly because the method is async.

Reply:
You get a weird liniting error if you do that. eslint likes consistent return values...